docs/index.md

```diff
@@ -1265,15 +1265,16 @@ Initiate the server setup process for a newly started server.
 
 ### Inputs
 
-| Param            | Type   | Description                                                  |
-| ---------------- | ------ | ------------------------------------------------------------ |
-| nics             | Object | Nic parameters to update                                     |
-| postsetup_script | String | Script to run after setup has completed                      |
-| hostname         | String | Hostname to set for the specified server                     |
-| disk_spares      | String | See `man disklayout` spares                                  |
-| disk_width       | String | See `man disklayout` width                                   |
-| disk_cache       | String | See `man disklayout` cache                                   |
-| disk_layout      | String | See `man disklayout` type      (single, mirror, raidz1, ...) |
+| Param             | Type   | Description                                                  |
+| ----------------- | ------ | ------------------------------------------------------------ |
+| nics              | Object | Nic parameters to update                                     |
+| postsetup_script  | String | Script to run after setup has completed                      |
+| hostname          | String | Hostname to set for the specified server                     |
+| disk_spares       | String | See `man disklayout` spares                                  |
+| disk_width        | String | See `man disklayout` width                                   |
+| disk_cache        | String | See `man disklayout` cache                                   |
+| disk_layout       | String | See `man disklayout` type      (single, mirror, raidz1, ...) |
+| encryption_enabled| String | See See `man mkzpool -e`                                     |
```

Comment:
Double 'See See' in the description line: See See `man mkzpool -e`

Reply:
Good catch!